packages/core/src/presets.ts

```diff
@@ -40,3 +40,17 @@ export const uploadthing = () =>
     },
     runtimeEnv: process.env,
   });
+
+/**
+ * Supabase Environment Variables
+ *   @see https://supabase.com/docs/guides/getting-started/quickstarts/nextjs
+ */
+
+export const supabase = () =>
+  createEnv({
+    server: {
+      NEXT_PUBLIC_SUPABASE_URL: z.string().optional(),
+      NEXT_PUBLIC_SUPABASE_ANON_KEY: z.string().optional(),
+    },
```

Comment:
Since these are NEXT_PUBLIC_, these should probably be client-variables

Reply:
okay, my bad